File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.java
import java.io.*;

public class Main{
    public static void main(String[] args) throws IOException{

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int T = Integer.parseInt(br.readLine());

        for (int i=0; i<T; i++) {
            String[] input = br.readLine().split(" ");
            int H = Integer.parseInt(input[0]);
            int W = Integer.parseInt(input[1]);
            int N = Integer.parseInt(input[2]);

            int floor, room;
            room = (N-1) / H + 1;

            if (N % H == 0) {
                floor = H;
            } else {
               floor = N % H;
            }

            bw.write((floor*100+room)+"\n");
        }

        bw.flush();
    }
}

File: 백준/Bronze/10250. ACM 호텔/ACM 호텔.py
if __name__ == '__main__':
    T = int(input())

    for _ in range(T):

        H, W, N = map(int, input().split())

        if N%H == 0:
            floor = H
        else:
            floor = N%H

        room = (N-1) // H + 1

        print(floor, end = "")
        if room < 10:
            print("0", end="")

        print(room)